Makefile

```diff
@@ -6,8 +6,13 @@ juno: vm ## compile
 	@mkdir -p build
 	@go build -a -ldflags="-X main.Version=$(shell git describe --tags)" -o build/juno ./cmd/juno/
 
+ifeq ($(VM_DEBUG),true)
```

Comment:
should we make this a global swtich and build juno in debug mode as well?

Reply:
Discussed in Slack: there are no debug/release builds for go